packages/devtools_app/test/integration_tests/server_test.dart

```diff
@@ -313,7 +313,9 @@ Future<Map<String, dynamic>> _waitForClients({
                   .any((c) => c['hasConnection'] == requiredConnectionState));
     },
     timeout: const Duration(seconds: 10),
-    timeoutMessage: 'Server did not return any known clients',
+    timeoutMessage: 'Server did not return any known clients'
```

Comment:
Heh, I tried to format this multiple times. `dartfmt` unwraps it like this 😄 I think it's because of the adjacent strings but it didn't really look great the other way either:

```dart
    timeoutMessage: 'Server did not return any known clients' +
        (requiredConnectionState != null
            ? (requiredConnectionState
                ? ' that are connected'
                : ' that are not connected')
            : '') +
        (true != null ? ' that are on page $requiredPage' : ''),
```

I'll see if I can shuffle it around a bit.

Reply:
```dart
true != null
```

Uh, that's certainly a bug :)